src/imports.js

```diff
@@ -29,7 +29,8 @@ const importRegex = new RegExp(
 export const fixShorthandImports = code => {
   return code.replaceAll(importRegex, found => {
     if (found.indexOf(" from") !== -1) return found
-    const whatMatch = found.matchAll(/"?([^"\s]+)"?,?\s*?/giu)
```

Comment:
alternatively could change the regex to always expect double-quotes

`/"([^"\s]+)",?\s*?/giu`

I believe the addition of optionals here in the last minute is what caused this issue 
short-hand imports should always be double quoted anyway, AFAIK

Reply:
changing to that (and adding an optional whitespace pre-comma) as it's more elegant